make_profiler/report/script.js

```diff
@@ -0,0 +1,148 @@
+function filterTarget() {
+    var input, filter, table, tr, td, i, txtValue;
+    input = document.getElementById("myInput");
+
+    filter = input.value.toUpperCase().split(" ").join("_");
+    table = document.getElementById("statusTable");
+    tr = table.getElementsByTagName("tr");
+
+    if (filter.startsWith('"') && filter.endsWith('"')) {
+        filter = filter.substring(1, filter.length - 1)
+        for (i = 0; i < tr.length; i++) {
+            td = tr[i].getElementsByTagName("td")[0];
+            if (td) {
+                txtValue = td.childNodes[0].textContent || td.innerText;
+                if (txtValue.toUpperCase() === filter) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    } else {
+        for (i = 0; i < tr.length; i++) {
+            td = tr[i].getElementsByTagName("td")[0];
+            if (td) {
+                txtValue = td.textContent || td.innerText;
+                if (txtValue.toUpperCase().indexOf(filter) > -1) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    }
+}
+
+//refresh page every 5 minutes
+setTimeout(function () {
+    window.location.reload(1);
+}, 1000 * 5 * 60);
+
+getStatus("report.json").then((resp) => {
+    let recs = resp.status;
+    let pipeline = resp.pipeline;
+    var pipelineTable =
```

Comment:
not that critical, but still 
https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994